Allure.SpecFlowPlugin/Allure.SpecFlowPlugin.csproj

```diff
@@ -26,7 +26,8 @@
 
     <ItemGroup>
         <PackageReference Include="CsvHelper" Version="30.0.1" />
-        <PackageReference Include="Lib.Harmony" Version="2.3.0-prerelease.2" />
+        <PackageReference Include="Lib.Harmony" Version="2.3.0-prerelease.5" />
+        <PackageReference Include="MonoMod.Utils" Version="25.0.4" />
```

Comment:
Harmony 2.3.0 already includes MonoMod.Utils v25.0.4+. We can remove this line now. Same for xUnit